apps/rush-lib/src/rush.schema.json

```diff
@@ -70,6 +70,17 @@
       },
       "additionalProperties": false
     },
+    "repository": {
+      "description": "The repository location",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "The url of the repository",
```

Comment:
>"The url of the repository" [](start = 25, length = 27)

This documentation is not very helpful -- obviously the "repoistory.url" stores the URL of the repository.  :-)

Maybe you could explain:
- Why do we need this URL?
- When is it NOT needed?
- Is there any special format required?
- What happens if it is not provided?